### DISCUSSION TITLE
[HELP] charon.log received netlink error: Unknown device type (95)  failed to create XFRM interface 'xfrmi-test-1480'

### DISCUSSION THREAD

**Discussion by aigenisi-aliyi** [strongswan/strongswan#1947] [General]
getting SPD hash threshold failed: Not supported (95)                   
00[KNL] getting SPD hash threshold failed: Not supported (95)                   
00[KNL] received netlink error: Unknown device type (95)                        
00[KNL] failed to create XFRM interface 'xfrmi-test-1480' 

dpd_action = none                                                    
08[CFG]    start_action = start                                                 
08[CFG]    close_action = none

**Comment by aigenisi-aliyi** [comment]
feature CONTAINER_DECODE:PKCS7 in plugin 'pem' has unmet dependency: CONTAINER_DECODE:PKCS7

**Reply by aigenisi-aliyi** [reply]
[IKE] <n2n-1|1> queueing IKE_ESTABLISH task

**Reply by aigenisi-aliyi** [reply]
activating CHILD_CREATE task

**Comment by tobiasbrunner** [comment]
> 00[KNL] getting SPD hash threshold failed: Not supported (95)

Not really a problem unless you want to configure an SPD hash threshold (see [`spdh_thresh` in the docs](https://docs.strongswan.org/docs/5.9/config/strongswanConf.html#_charon_plugins_kernel_netlink)). Anyway, that seems to indicate an insanely old Linux kernel as this has been supported since 2.6.22, or that XFRM is completely disabled in that kernel (see [the docs for the required kernel modules](https://docs.strongswan.org/docs/5.9/install/kernelModules.html)).

> 00[KNL] received netlink error: Unknown device type (95)
> 00[KNL] failed to create XFRM interface 'xfrmi-test-1480'

Unless you actually need to use XFRM interfaces, you can just ignore these messages.

> feature CONTAINER_DECODE:PKCS7 in plugin 'pem' has unmet dependency: CONTAINER_DECODE:PKCS7

Nothing you have to concern yourself with.

**Reply by aigenisi-aliyi** [reply]
feature CONTAINER_DECODE:PKCS7 in plugin 'pem' has unmet dependency: CONTAINER_DECODE:PKCS7
